harness/determined/common/experimental/experiment.py

```diff
@@ -432,3 +432,25 @@ def _from_bindings(
         exp = cls(session=session, experiment_id=exp_bindings.id)
         exp._hydrate(exp_bindings)
         return exp
+
+
+class ExperimentReference(Experiment):
+    """A legacy class representing an Experiment object.
+
+    This class was renamed to :class:`~determined.experimental.Experiment` and will be removed
+    in a future release.
+    """
+
+    def __init__(
```

Comment:
nit: `__new__` is responsible for creating objects, not `__init__` (you can create a new `ExperimentReference` without calling `ExperimentReference.__init__`, but you can't create one without calling `__new__`). I think it's _slightly_ safer/more correct to override `__new__` for that reason.

Reply:
discussed offline. new is too confusing.